01-basics/01-CounterButtonApp/index.html

```diff
@@ -1,14 +1,17 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="UTF-8" />
-    <title>CounterButtonApp</title>
-  </head>
-  <body>
-    <div id="app">
-      <button type="button">0</button>
-    </div>
-
-    <script type="module" src="./script.js"></script>
-  </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>CounterButtonApp</title>
+</head>
+
+<body>
+  <div id="app">
+    <button @click="increment()" type="button">{{counter}}</button>
+  </div>
```

Comment:
It is common practice to add spaces in text interpolation: `{{ counter }}`

Reply:
Also, if we don't need to add any custom values to the event handler. we can pass the handler without calling `@click="increment"`. Then Vue won't wrapp it into a new function.